orchagent/pfcactionhandler.h

```diff
@@ -73,10 +73,12 @@ class PfcWdAclHandler: public PfcWdActionHandler
         // class shared dict: ACL table name -> ACL table
         static std::map<std::string, AclTable> m_aclTables;
```

Comment:
>m_aclTables [](start = 47, length = 11)

You may use 2 maps for acl tables. There is no benefit to mix ingress and egress into one.

Reply:
It would better to maintain one map for all aclTables from my perspective. The process of handling ingress/egress table are the same. Maintaining two maps every time looks redundant.